### ISSUE TITLE
Design updates

### ISSUE THREAD

**Issue by kylehalleman** [kylehalleman/covid19inwi#14]
The site currently uses a very minimalist theme based off of [a small CSS project I made](https://github.com/kylehalleman/walden/). I chose this because it has strong legibility, minimal footprint, and by using element selectors promotes accessibility (the color palette for both the light and dark themes also passes WCAG 2.1). It also includes a dark mode based on a user's OS settings.

Some updates to the site might be needed to further our goals for the site. I would only ask any updates stay inline with the following design goals.

## Design goals

* Accessible: color palette should pass [WCAG 2.1 AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) and semantic HTML should be used for any interface patterns.
* Responsive: should pass [WCAG 2.1 1.4.10 (Reflow)](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html).
* Performant: unless a good case is made, I do not see the need in adding web fonts, JavaScript, or excessively large images.

**Comment by AnnetteMKE** [comment]
It doesn't need to be this fancy for sure, but is there any reason not to copy the format Johns Hopkins is using, adapting some of the blocks and/or adding blocks with information about resources? 
https://coronavirus.jhu.edu/map.html

I think we should have some images to break up the copy (especially because I don't think we will have a heat map in the MKE site). 

Other than that, I agree - keep it super simple, super clean.

**Comment by kylehalleman** [comment]
While I find the Johns Hopkins dashboard incredibly useful, this site isn't trying to cover the same type of information they are. I do think there is value in creating multiple columns in the layout on larger screen sizes, that would help reduce the feeling of the page just being a very long set of links.

I agree we should make sections more distinct, but I'm not sure what the benefit of adding images would be. They add to the page weight and unless they provide real value outside of visual stimulation, I would err against them. However, if you find relevant and useful images that would make sense in certain sections or pages, please share them (if we would have the rights to use them, of course 🙂) in this issue.

@erinmagennis [mentioned in another issue](https://github.com/kylehalleman/covid19inwi/issues/15#issuecomment-607295955) separating out section more clearly like [this site does](https://socmilwaukee.org/index/coronavirus-information/). I like that direction Erin talked about and want to start there. Like you said, breaking up the copy will help with scanability.

I'll be working on some of these tweaks this afternoon and will ask for another round of review when that is up.

Thanks again for your input and feedback!
